src/django/composeexample/urls.py

```diff
@@ -15,6 +15,8 @@
 """
 from django.contrib import admin
 from django.urls import path, include
+## This seems to be unused -- it's a good idea to configure linting in your text editor to catch
+## these types of things so they can be removed.
```

Comment:
Add-on comment: we try to enforce linting and code formatting in CI, so although you may not receive review comments, formatting errors may cause the build for a PR to fail.

Reply:
Got it - do you usually catch things like unused imports thru manual review? Or are there linters that you regularly use when developing locally?

Reply:
No, we try to put as much onto tools as possible and not require manual effort for this kind of thing; I use a flake8 plugin for my editor which flags formatting and linting errors as I make changes.